### ISSUE TITLE
Unittest Segmentation Fault: VaapiPostProcessScalerTest.mapToRange

### ISSUE THREAD

**Issue by uartie** [01org/libyami#572]
On platforms that don't support `vaQueryVideoProcFilterCaps` (e.g. Ivybridge), the `VaapiPostProcessScalerTest.mapToRange` test will segfault.  This is because `caps` is a NULL pointer and we try to deference it.  Before dereferencing the pointer, there should be an ASSERT_TRUE(bool(caps)) before to ensure the test does not continue when it's NULL.

`$ ./vpp/unittest`

```
[==========] Running 5 tests from 1 test case.
[----------] Global test environment set-up.
[----------] 5 tests from VaapiPostProcessScalerTest
[ RUN      ] VaapiPostProcessScalerTest.Factory
[       OK ] VaapiPostProcessScalerTest.Factory (0 ms)
[ RUN      ] VaapiPostProcessScalerTest.mapToRange
libva info: VA-API version 0.39.3
libva info: va_getDriverName() returns 0
libva info: Trying to open /home/uartie/Work/workspace/media/install/lib/dri/i965_drv_video.so
libva info: Found init function __vaDriverInit_0_39
libva info: va_openDriver() returns 0
libyami error 5957 (vaapipostprocess_base.cpp, 84): vaQueryVideoProcFilterCaps: the requested filter is not supported
vaapipostprocess_scaler_unittest.cpp:129: Failure
Value of: scaler.mapToRange(value, level, minLevel, maxLevel, filterType, caps)
  Actual: false
Expected: true
vaapipostprocess_scaler_unittest.cpp:130: Failure
Value of: bool(caps)
  Actual: false
Expected: true
Segmentation fault (core dumped)
```

**Comment by xuguangxin** [comment]
Hi @uartie ,
We will check it after Q3 release

**Comment by xuguangxin** [comment]
fixed by 	3376fbf
